# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/valentin/asterisk/linux_ixjuser_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./channels/chan_phone.c.orig	2004-11-28 13:39:33.005340536 +0100
+++ ./channels/chan_phone.c	2004-11-28 13:39:57.254654080 +0100
@@ -32,6 +32,7 @@
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <linux/telephony.h>
+#include <linux/compiler.h>
 /* Still use some IXJ specific stuff */
 #include <linux/ixjuser.h>
 #include "DialTone.h"
